src/components/ProfileEditEmailPage.js

```diff
@@ -86,11 +88,10 @@ const ProfileEditEmailPage = () => {
           fullWidth
           label="New email"
           name="email"
-          placeholder="unicorn@usepow.app"
           autoComplete="email"
           helperText={
             <>
-              You current email address is <strong>test@test.com</strong>.
+              Your current POW! email is <strong>yo@POW.app</strong>.
```

Comment:
If you change this back (the whole file really) to exactly the same as it was we might not get any conflicts with the changes you made in the bug fix Pull Request.

Reply:
Question:
WHEN do you mean by "...to exactly the same as it was..."? 
Before this pull request?

Ok I copy paste the  ProfileEditEmailPage.js code from the "master" branch